OpenMarket/OpenMarket/Resource/Model/Networking/APIType.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  APIType.swift
+//  OpenMarket
+//
+//  Copyright (c) 2022 Minii All rights reserved.
+
+import Foundation
+
+protocol APIType {
+    var baseURL: String { get }
+    var path: String { get }
+    var params: [String: String] { get }
```

Comment:
```suggestion
    var parameters: [String: String] { get }
```
간결한것보다 길어지더라도 명확해지는 것이 요즘 네이밍 추세로 보입니다 ㅎㅎ

Reply:
구: param, img, ..
현: parameter, image